src/OrchardCore/OrchardCore.Search.Elasticsearch.Abstractions/ElasticsearchOptions.cs

```diff
@@ -7,4 +7,6 @@ public class ElasticsearchOptions
     public string IndexPrefix { get; set; }
 
     public Dictionary<string, JsonObject> Analyzers { get; } = [];
+
+    public Dictionary<string, JsonObject> Filter { get; } = [];
```

Comment:
Filters

Reply:
![image](https://github.com/user-attachments/assets/23bb1e5e-ef0e-480b-9401-68af70106b2a)

I think we should left name as "Filters", because it is how this section called in elasticsearch terminology.

https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-custom-analyzer.html
Settings.Analysis.FIlter and there listed custom token filters

Reply:
hmm, ok, that's weird but I guess that we can live with it.